### ISSUE TITLE
Create Risk Maps crashes after completion

### ISSUE THREAD

**Issue by jessnoo** [Esri/transportation-safety-tools#44]
After running Create Risk Maps, AppROT crashes

Versions affected: 10.4 (PreRelease, 5467), 10.2.2

Repro steps:

1. Complete Basic Segmentation and Assign Crashes steps
2. In ArcMap, run Create Risk Maps tool
3. Tool finishes successfully and new mxd is opened
4. Close new mxd without saving
5. Close the tool dialog (this tool always runs in the foreground)

>> receive message: 

![image](https://cloud.githubusercontent.com/assets/6181441/11981822/0107e3a0-a95b-11e5-9417-a1985e622f17.png)

**Comment by jessnoo** [comment]
none of the other versions pop the resulting map open -- is that a behavior we want?

**Comment by chris-fox** [comment]
@jessnoo so I found opening the new mxd after the tool finishes was added as part of #34. I also created a simple script tool that takes no parameters and all it executes is the single line of code below to open a mxd. This also reproduces the crash.

os.system(r"C:\test\RiskMaps_Template.mxd")

So it looks like an ArcMap/AppRot bug, specifically when opening arcmap in a gp process. I tried other methods for opening an mxd, subprocess.create(), os.createfile(), both resulted in the same crash. Interestingly I couldn't repro the crash if I ran the tool in the background.

You saw this on 10.4 as well? I couldn't repro on 10.3.1.
